sevenseconds/config/vpc.py

```diff
@@ -788,6 +788,11 @@ def if_stups_tool(ni: dict):
             for gateway in ec2c.describe_nat_gateways()['NatGateways']:
                 if gateway.get('NatGatewayAddresses', {})[0].get('AllocationId') == allocation_id:
                     return True
+
+        # KMS VPC Endpoint
+        group_name = ','.join([group["GroupName"] for group in ni.get("Groups")])
+        if group_name == "KMS VPC Endpoint":
```

Comment:
If the KMS has another SG it won't be filtered. I mean, if we change it here, we will have to remember to update it here too, which shouldn't be a problem. But if changed manually it will be a problem, so what do you think about it?

```python
if "KMS VPC Endpoint" in group_name:
```

Reply:
actually, it would just prevent if we add another SG, so I don't think we have to worry about it. Just ignore this one.

Reply:
I think it's better to check exactly what we expect and then fail if we get something else. Then the human can validate why it has more SGs than expected before deleting everything.